### PR TITLE
Empty docsummary output for undefined value.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review
@baldersheim : FYI

### CODE REVIEW

**Review by geirst — APPROVED** [review]
👍

File: searchsummary/src/vespa/searchsummary/docsummary/check_undefined_value_visitor.cpp [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+void
+CheckUndefinedValueVisitor::visit(const document::Document&)
+{
+}
+
+void
+CheckUndefinedValueVisitor::visit(const document::DoubleFieldValue& value)
+{
+    if (isUndefined(value.getValue())) {
```
Comment (baldersheim) [inline searchsummary/src/vespa/searchsummary/docsummary/check_undefined_value_visitor.cpp:46]: Why is this necessary ?
Single value fields are only undefined if they do not exist.
The special undefined value is only used in attributes, and should hence never make it into any document.

Comment (geirst) [reply searchsummary/src/vespa/searchsummary/docsummary/check_undefined_value_visitor.cpp:46]: This has been resolved in https://github.com/vespa-engine/vespa/pull/23317.
